Gestalt/ThemeManager.swift

```diff
@@ -262,3 +264,17 @@ extension ThemeManager: CustomStringConvertible {
         }
     }
 }
+
+#if os(iOS)
+open class SafeApplication {
```

Comment:
Is there a need to make this `open`? I'd rather keep it an `internal` implementation detail, if possible.
Or maybe just move it into `ThemeManager` as a lazy var, so we only need to do the dynamic dispatch dance once per app launch?

```swift
extension ThemeManager {
    internal lazy var sharedApplication: UIApplication? = {
        let selector = NSSelectorFromString("sharedApplication")
        guard UIApplication.responds(to: selector) else {
            // Extensions cannot access UIApplication
            return nil
        }
        let shared = UIApplication.perform(selector)
        return shared?.takeUnretainedValue() as? UIApplication
    }
}()
```

Reply:
Oh yeah, I haven't given much thought to access control. It's just open because I accessed it from different contexts to debug.
I can certainly put it into a private stored property. I will do some checks and push an update to this PR if there are no problems. I think we can't put a stored property into an extension though.
Also is there any particular reason why you want to make it `lazy var` instead of just a `let` constant?